feature-libs/organization/unit-order/styles/components/unit-level-order-history/_unit-level-order-history.scss

```diff
@@ -0,0 +1,472 @@
+
+    @include media-breakpoint-down(sm) {
+      padding-top: 40px;
+      padding-inline-end: 20px;
+      padding-bottom: 0;
+      padding-inline-start: 20px;
+    }
+  }
+
+  .cx-unit-level-order-history-thead-mobile {
+    @include media-breakpoint-down(sm) {
+      display: none;
+    }
+  }
+
+  .cx-unit-level-order-history-code {
+    .cx-unit-level-order-history-value {
+      text-decoration: underline;
+    }
+  }
+
+  .cx-unit-level-order-history-placed {
+    text-align: center;
+
+    @include media-breakpoint-down(sm) {
+      text-align: start;
+    }
+  }
+
+  .cx-unit-level-order-history-unit,
+  .cx-unit-level-order-history-buyer,
+  .cx-unit-level-order-history-status {
+    text-align: center;
+
+    @include media-breakpoint-down(sm) {
+      text-align: start;
+    }
+  }
+
+  .cx-unit-level-order-history-total {
+    text-align: end;
+
+    @include media-breakpoint-down(sm) {
+      text-align: start;
+    }
+  }
+
+  .cx-unit-level-order-history-label {
+    text-transform: uppercase;
+    color: var(--cx-color-secondary);
+
+    @include media-breakpoint-up(md) {
+      display: none;
+    }
+
+    @include media-breakpoint-down(sm) {
+      @include type('8');
+      min-width: 110px;
+    }
+  }
+
+  .cx-unit-level-order-history-value {
+    color: var(--cx-color-text);
+
+    @include media-breakpoint-down(sm) {
+      @include type('5');
+      font-weight: $font-weight-normal;
+    }
+  }
+
+  .cx-unit-level-order-history-form-group {
+    padding: 0;
+    margin-bottom: 0;
+    align-items: center;
+    display: flex;
+    width: 40%;
+
+    span {
+      margin-inline-end: 0.5rem;
+      flex: none;
+    }
+
+    cx-sorting {
+      width: 100%;
+    }
+
+    @include media-breakpoint-down(sm) {
+      padding: 1.25rem;
+      width: 100%;
+    }
+  }
+
+  .cx-unit-level-order-history-sort {
+    align-items: center;
+
+    &.top {
+      display: flex;
+      justify-content: space-between;
+      padding: 1rem 0;
+      margin: 0;
+
+      @include media-breakpoint-down(sm) {
+        flex-direction: column;
+        align-items: flex-start;
+        padding-top: 0;
+      }
+    }
+
+    &.bottom {
+      display: flex;
+      justify-content: space-between;
+      padding: 1rem 0 1rem 0;
+      margin: 0;
+
+      @include media-breakpoint-down(sm) {
+        flex-direction: column;
+        align-items: flex-start;
+      }
+    }
+  }
+
+  .cx-unit-level-order-history-pagination {
+    margin-inline-start: auto;
+    @include media-breakpoint-down(sm) {
+      align-items: flex-start;
+      padding-inline-start: 1.25rem;
+      margin-inline-start: 0;
+    }
+  }
+
+  .cx-unit-level-order-history-total-result {
+    font-style: italic;
+    color: var(--cx-color-secondary);
+    padding-inline-start: 1.25rem;
+    @include media-breakpoint-down(sm) {
+      padding-bottom: 1rem;
+    }
+
+    cx-total {
+      display: flex;
+      column-gap: 0.5rem;
+    }
+  }
+
+  .cx-vertical-line-separator {
```

Comment:
`.cx-vertical-line-separator` seems like a class that you might reuse on another element. If an element needs a border or styling we should add the styling to it by adding a class to the element if it doesn't have one already. You should remove this and add the border to the specific html element with a class name following `cx-` naming.